lib/worker/index.js

```diff
@@ -145,12 +147,12 @@ module.exports = {
   queues,
 
   start() {
-    queues.discovery.process(5, commonMiddleware(discovery(app, queues)));
-    queues.installation.process(Number(CONCURRENT_WORKERS), commonMiddleware(processInstallation(app, queues)));
-    queues.push.process(Number(CONCURRENT_WORKERS), commonMiddleware(processPush(app)));
+    queues.discovery.process(5, commonMiddleware(discovery(queues)));
+    queues.installation.process(Number(CONCURRENT_WORKERS), commonMiddleware(processInstallation(queues)));
+    queues.push.process(Number(CONCURRENT_WORKERS), commonMiddleware(processPush()));
     queues.metrics.process(1, commonMiddleware(metricsJob));
 
-    app.log(`Worker process started with ${CONCURRENT_WORKERS} CONCURRENT WORKERS`);
+    logger.info(`Worker process started with ${CONCURRENT_WORKERS} CONCURRENT WORKERS`);
```

Comment:
Same. I don't think app.log.info is the function in app.log. Can you just keep it app.log?

Reply:
@AmrutaKawade We don't have app here , so using logger.info.